index.html

```diff
@@ -103,6 +103,26 @@ <h1>Lambda Times</h1>
       </div>
     </div>
 
+    <div class="card" data-tab="technology">
+      <div class="headline">VR: How will society change?</div>
+      <div class="author">
+        <div class="img-container">
+          <img src="./assets/max.jpg" />
+        </div>
+        <span>By MAX GOODBOYE</span>
+      </div>
+    </div>
+
+    <div class="card" data-tab="bootstrap">
+      <div class="headline">WTF is bootstrap even?</div>
+      <div class="author">
+        <div class="img-container">
+          <img src="./assets/max.jpg" />
+        </div>
+        <span>By MAX GOODBOYE</span>
+      </div>
+    </div>
+
     <div class="card" data-tab="jquery">
       <div class="headline">History of Javascript Libraries: from jQuery to React</div>
```

Comment:
- Inaki You did great job on the sprint challenge. You properly implemented Tabs and Cards components into the project and were able to filter out articles by selecting particular Tab. You also added new articles to test Your components which concludes the MVP requirements 👍

- Overall:
* Tabs and Cards properly implemented (MVP)
* New Articles added (MVP)
* Good answers to questions (MVP)

- For possibile improvements try and have more commits messages just in case something goes wrong it the code 
- I can see that You are continually improving .Nice job, You displayed a good understanding on the subject 👍